portfolio/src/main/java/com/google/sps/servlets/CreateNewComment.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.gson.Gson;
+import com.google.sps.servlets.models.Comment;
+
+@WebServlet("/create-new-comment")
+public class CreateNewComment extends HttpServlet {  
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String username = request.getParameter("username"); // get the username from the form
+    String commentContent = request.getParameter("comment"); // get the comment from the form
+
+    response.setContentType("text/html"); // set response type
+
+    Entity commentEntity = Comment.createNewCommentEntity(username, commentContent);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService(); // create instance of DatastoreService class
+    datastore.put(commentEntity);
+    
+    response.sendRedirect("templates/comments.html"); // redirect back to the HTML page
```

Comment:
Because I don't render the new comment element inside the onclick event of the "post comment" button, I need to refresh for the page to update the changes. I aim to implement live-updating eventually, but for now I think the redirect makes sense.